docs/build.sh

```diff
@@ -44,6 +44,9 @@ pipenv install
 pipenv run flowmachine &
 echo "Started FlowMachine."
 pipenv run quart run --port 9090 &
+echo "Retrieving API spec"
+sleep 5
+curl http://localhost:9090/api/0/spec/openapi-redoc.json -o source/_static/openapi-redoc.json
```

Comment:
I'm not entirely sure why, but somehow sleeping and retrieving the spec from the actual running API server feels a little awkward/brittle to me. How about copying the json spec from the `.approved.txt` file in the flowapi tests instead?

Reply:
Approval tests need the version stripped out, which makes the result not a valid spec :(

Reply:
What if we use a more stable version (e.g. the tagged release `0.5.2` rather than the full version `0.5.2+113.g1d4132f0`) for the API, and don't strip it out in the approval tests?

Or alternatively put the version back in here manually from the `.approved.txt` file instead of retrieving it from the API server directly?